provider/lxd/environ_test.go

```diff
@@ -184,15 +184,15 @@ func (s *environSuite) TestDestroyController(c *gc.C) {
 	s.Stub.CheckCalls(c, []gitjujutesting.StubCall{
 		{"Destroy", []interface{}{s.callCtx}},
 		{"StorageSupported", nil},
-		{"StoragePools", nil},
-		{"VolumeList", []interface{}{"juju"}},
-		{"VolumeList", []interface{}{"juju-zfs"}},
+		{"GetStoragePools", nil},
+		{"GetStoragePoolVolumes", []interface{}{"juju"}},
+		{"GetStoragePoolVolumes", []interface{}{"juju-zfs"}},
 		{"AliveContainers", []interface{}{"juju-"}},
 		{"RemoveContainers", []interface{}{[]string{machine1.Name}}},
 		{"StorageSupported", nil},
-		{"StoragePools", nil},
-		{"VolumeList", []interface{}{"juju"}},
-		{"VolumeDelete", []interface{}{"juju", "ours"}},
-		{"VolumeList", []interface{}{"juju-zfs"}},
+		{"GetStoragePools", nil},
+		{"GetStoragePoolVolumes", []interface{}{"juju"}},
+		{"DeleteStoragePoolVolume", []interface{}{"juju", "custom", "ours"}},
+		{"GetStoragePoolVolumes", []interface{}{"juju-zfs"}},
```

Comment:
I'm assuming all of these are just changing from what it used-to-be-called when we had our own shims to what it is called now in the official new LXD api. I haven't validated that these are correct renames. I'm slightly concerned since we're mocking it out, but I'm trusting that you've done appropriate due-diligence and done the right thing here.

(note though, that probably lxd containers + storage is probably not in our normal CI suite, although it probably should be.)

Reply:
Yes. for most of these, the wrapping methods (old names) were superfluous and were wrapping these exact calls, so those are removed and replaced with signatures exposed by LXD (called through our server via embedding).

I have implemented a couple of creation methods on the new server to abstract away dealing with the LXD object types, but the others are direct to LXD as you note.

I am confident that this has been correctly done.